README.md

```diff
@@ -22,7 +22,7 @@ Community maintained list of Algorithm and Data Structure implementations.
 | Algorithm                        | C                  | CPP                | Java               | Python2            | Python3            | Golang | JavaScript | C#    | TypeScript |
 | :---                             | :---:              | :---:              | :---:              | :---:              | :---               | :---:  | :---:      | :---: | :---:      |
 | Bin Sort                         |                    |                    |                    |                    |                    |        |            |       |            |
-| Binary Search                    | :white_check_mark: | :white_check_mark: | :white_check_mark: |                    | :white_check_mark: |        |            |       |            |
+| Binary Search                    | :white_check_mark: | :white_check_mark: | :white_check_mark: |                    | :white_check_mark: |        |:white_check_mark:|       |            |
```

Comment:
Codacy found an issue: [[table-cell-padding] Cell should be padded](https://app.codacy.com/gh/FOSS-UCSC/FOSSALGO/pullRequest?prid=6275998)